[2022-08-16 17:22:01,351] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: yandex-rates-test.insert_data manual__2022-08-16T17:21:29.666636+00:00 [queued]>
[2022-08-16 17:22:01,586] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: yandex-rates-test.insert_data manual__2022-08-16T17:21:29.666636+00:00 [queued]>
[2022-08-16 17:22:01,634] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-16 17:22:01,666] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2022-08-16 17:22:01,683] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-16 17:22:01,789] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): insert_data> on 2022-08-16 17:21:29.666636+00:00
[2022-08-16 17:22:01,803] {standard_task_runner.py:52} INFO - Started process 2454 to run task
[2022-08-16 17:22:01,824] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'yandex-rates-test', 'insert_data', 'manual__2022-08-16T17:21:29.666636+00:00', '--job-id', '70', '--raw', '--subdir', 'DAGS_FOLDER/yandex_dag.py', '--cfg-path', '/tmp/tmpwbhldszc', '--error-file', '/tmp/tmpjzjnmzlt']
[2022-08-16 17:22:01,825] {standard_task_runner.py:80} INFO - Job 70: Subtask insert_data
[2022-08-16 17:22:02,163] {task_command.py:371} INFO - Running <TaskInstance: yandex-rates-test.insert_data manual__2022-08-16T17:21:29.666636+00:00 [running]> on host c6d8e67217b1
[2022-08-16 17:22:02,419] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=yandex-rates-test
AIRFLOW_CTX_TASK_ID=insert_data
AIRFLOW_CTX_EXECUTION_DATE=2022-08-16T17:21:29.666636+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-08-16T17:21:29.666636+00:00
[2022-08-16 17:22:02,474] {logging_mixin.py:115} INFO - rate_date:  2022-08-16
[2022-08-16 17:22:02,475] {logging_mixin.py:115} INFO - value_:  24207.92857100000037
[2022-08-16 17:22:02,505] {base.py:68} INFO - Using connection ID 'postgre_test_db' for task execution.
[2022-08-16 17:22:02,578] {python.py:173} INFO - Done. Returned value was: None
[2022-08-16 17:22:02,620] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=yandex-rates-test, task_id=insert_data, execution_date=20220816T172129, start_date=20220816T172201, end_date=20220816T172202
[2022-08-16 17:22:02,745] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-16 17:22:03,231] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
